src/targets/shell/curl.js

```diff
@@ -39,6 +40,22 @@ module.exports = function (source, options) {
     code.push(opts.short ? '-0' : '--http1.0')
   }
 
+  // if multipart form data, we want to remove the boundary
+  if (source.postData.mimeType === 'multipart/form-data') {
+    const contentTypeHeaderName = headerHelpers.getHeaderName(source.headersObj, 'content-type')
+    const contentTypeHeader = source.headersObj[contentTypeHeaderName]
+
+    if (contentTypeHeaderName && contentTypeHeader) {
+      // remove the leading semi colon and boundary
+      // up to the next semi colon or the end of string
+      const noBoundary = contentTypeHeader.replace(/; boundary.+?(?=(;|$))/, '')
```

Comment:
I double checked these regex and functions as expected.

<img width="776" alt="Screen Shot 2021-09-03 at 11 16 37 AM" src="https://user-images.githubusercontent.com/5934780/132051951-09a29744-dd4c-4a38-b926-71434bcad4ed.png">